app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      @user = current_user
+      # binding.pry
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets/new' do
```

Comment:
Typically, tweet creation would be posted to `/tweets` instead of `/tweets/new`.